### DISCUSSION TITLE
FLUX

### DISCUSSION THREAD

**Discussion by bsdesigner** [lllyasviel/Fooocus#3446] [General]
Will Fooocus be able to work with FLUX models?

https://civitai.com/models/618692/flux
https://huggingface.co/black-forest-labs/FLUX.1-dev
https://blackforestlabs.ai/announcing-black-forest-labs/

**Comment by jkvaman** [comment]
Flux is now working on WebUI Forge.

**Comment by BGT77** [comment]
Flux is working with Swarm - and can be installed at the same time as Swarm installation.

**Reply by bsdesigner** [reply]
Yes, Flux is working with Comfy, Swarm & Forge. But it was about the possibility of bolting the Flux to the Fooocus. Now this question is no longer relevant...
